Ports/openssh/package.sh

```diff
@@ -12,7 +12,6 @@ configopts=("--prefix=/usr/local" "--disable-utmp" "--disable-strip" "--sysconfd
 export LDFLAGS="-lcrypt -lcore"
 
 pre_configure() {
-    ensure_new_config_sub
```

Comment:
This is only redundant *after* your other commit, so please put this commit after that.

Reply:
My bad, rebased to change order. Should be fine now.